src/App.tsx

```diff
@@ -1,7 +1,117 @@
-import { greet } from "./utils/greet";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import { getSummary } from "./utils/getSummary";
 
-function App(): JSX.Element {
-  return <h1>{greet("World")}</h1>;
-}
+export default function App(): JSX.Element {
+  interface entry {
+    entry_id: number;
+    title_text: string;
+    summary_text: string;
+    time: string;
+  }
+  const [tenPastes, setTenPastes] = useState<entry[]>([]);
+  const [title, setTitle] = useState<string>("");
+  const [summary, setSummary] = useState<string>("");
+  const [clickedPaste, setClickedPastes] = useState<entry[]>([]);
+  const [isPasteClicked, setIsPasteClicked] = useState<boolean>(false);
+  const [newPaste, setNewPaste] = useState<entry[]>([]);
+
+  function handleSubmit() {
+    console.log("would send text");
+    //gather detail of new entry
+    const newEntry = {
+      title: title,
+      summary: summary,
+    };
+    //send an HTTP post request to our API server with new entry
```

Comment:
Really enjoy your comments - they have been useful in understanding your intent behind things